CONTRIBUTING.md

```diff
@@ -37,17 +141,31 @@ In the terminal on your computer:
 2. Click <kbd>Install</kbd>
 3. Follow the instructions to select your fork
 
-### Opening a PR
-
-One you have made your changes, you’re ready to create a “Pull Request”! This will let the Astro docs team know you have some changes to propose. At this point we can give you feedback and might request changes. In general, we like to have at least one other person who knows the language you are translating into review the PR.
 
-[Read more about making a pull request in GitHub’s docs](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-a-pull-request)
+-----
 
 ## Style Guide
 
-We are developing a full Style Guide to help our contributors provide new content with a consistent style and voice! For now, here are some specific items you should know about when writing new docs content.
+We are developing a full Style Guide to help our contributors provide new content with a consistent style and voice! 
+
+### Tone
+
+As a general guide for writing tone, you can follow the [Google Developers Guide](https://developers.google.com/style/tone):
+
+"In your documents, aim for a voice and tone that's conversational, friendly, and respectful without being overly colloquial or frivolous; a voice that's casual and natural and approachable, not pedantic or pushy. Try to sound like a knowledgeable friend who understands what the developer wants to do.
 
-## Writing Asides (aka how not to abuse `blockquote`)
+Don't try to write exactly the way you speak; you probably speak more colloquially and verbosely than you should write, at least for developer documentation. But aim for a conversational tone rather than a formal one.
```

Comment:
```suggestion
> Don't try to write exactly the way you speak; you probably speak more colloquially and verbosely than you should write, at least for developer documentation. But aim for a conversational tone rather than a formal one.
```